### DISCUSSION TITLE
Band Contrast not exporting from Aztec Crystal

### DISCUSSION THREAD

**Discussion by naomijoycerodgers** [mtex-toolbox/mtex#2157] [Ask Anything]
Hi all,

New mtex user here.  My data was collected using Aztec, and I was able to export a .ctf file from aztec crystal.  However, it appears that band contrast was not exported with the rest of the data (when I type ebsd.[tab] BC or band contrast are not one of the options).  Does anyone have advice or suggestions?

Thanks in advance for any help!
Naomi

**Comment by kilir** [comment]
Hi

it should be `ebsd.bc` I think. You could also inspect `ebsd.prop`.

Cheers,
Rüdiger
